routes/favicon.coffee

```diff
@@ -0,0 +1,28 @@
+request = require 'request'
+fs = require 'fs'
+cachefile = 'favicon.ico'
+
+favicon = (req, res, next) ->
+	try
+		res.send fs.readFileSync cachefile
```

Comment:
While using the async variants, I replaced this with the more direct `res.sendFile`. When accessed through Firefox, it shows up like any ordinary image would, so I trust the library on this one.